src/routes/collectibles/handlers.rs

```diff
@@ -26,13 +26,13 @@ pub async fn collectibles(
         trusted.unwrap_or(false),
         exclude_spam.unwrap_or(true)
     )?;
-
-    Ok(RawJson(
-        RequestCached::new_from_context(url, &context, ChainCache::from(chain_id))
-            .request_timeout(collectibles_request_timeout())
-            .execute()
-            .await?,
-    ))
+    let body = RequestCached::new_from_context(url, &context, ChainCache::from(chain_id))
+        .request_timeout(collectibles_request_timeout())
+        .execute()
+        .await?;
+    Ok(Json(serde_json::from_str::<Vec<ServiceCollectible>>(
```

Comment:
This doesn't make any sense. We first serialize them as a string and then convert them back to structs to convert them back to strings. 

With our current caching we cannot use the `Json` type.

Reply:
Just to provider better documentation I would not add this additional parsing overhead